src/childinfo.c

```diff
@@ -89,11 +91,17 @@ void sendChildInfoGeneric(childInfoType info_type, size_t keys, double progress,
         cow = zmalloc_get_private_dirty(-1);
         cow_updated = getMonotonicUs();
         cow_update_cost = cow_updated - now;
-
-        if (cow) {
-            serverLog((info_type == CHILD_INFO_TYPE_CURRENT_INFO) ? LL_VERBOSE : LL_NOTICE,
-                      "%s: %zu MB of memory used by copy-on-write",
-                      pname, cow / (1024 * 1024));
+        if (cow > peak_cow) peak_cow = cow;
+        sum_cow += cow;
+        update_count++;
+
+        int cow_info = (info_type != CHILD_INFO_TYPE_CURRENT_INFO);
+        if (cow || cow_info) {
+            serverLog(cow_info ? LL_NOTICE : LL_VERBOSE,
```

Comment:
Hi @oranagra For the log, i think maybe users collect CoW info by this log, I break old format, WDYT? And currently I am not satisfied with current description, do you have advise?

Reply:
i don't think we need to retain backward compatibility with log messages.
maybe ins some special cases like redis fails to start up due to bind address being used and such, in case someone (like our tests) attempts to resolve that and retry.

But breaking this log message won't result in any actual problem, just missing metrics, and the right way to extract these metrics is from INFO (always has been)